ivy/functional/backends/jax/experimental/linear_algebra.py

```diff
@@ -168,7 +168,7 @@ def dot(
     *,
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
-    return jnp.dot(a, b, out=out)
+    return jnp.dot(a, b)
```

Comment:
It seems that there are a lot of problems with `ivy.dot` function. Would  best to create a Trello card for this first and then fix all of these in a separate PR.